target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8074-deco-x80-5g.dts

```diff
@@ -0,0 +1,488 @@
+/* SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/* Copyright (c) 2022, Robert Marko <robimarko@gmail.com> */
```

Comment:
Please look into how multiline comments work, there is no point in opening and closing comments for each line.